### PR TITLE
adding to the bottombar the new icon options

### PR DESCRIPTION
**Pull request by sombriks**
Hello!

This pull request intends to expose to the bottom bar the new options available to use as source of the icons.

Please review and provide feedback, i'll kindly attend any request which makes this improvement find his way.

For your time thank you.

### CODE REVIEW

File: docs/src/pages/components/BottomBar.vue [hunk 1]
```diff
@@ -68,6 +68,18 @@
               </md-table-row>
 
               <md-table-row>
+                <md-table-cell>md-src</md-table-cell>
```
Comment (marcosmoura) [inline docs/src/pages/components/BottomBar.vue]: I think that `md-icon-src` will be more clear. `md-src` could be related to the link src on the bottom bar.

File: src/components/mdBottomBar/mdBottomBarItem.vue [hunk 2]
```diff
@@ -1,6 +1,9 @@
 <template>
   <a :href="href" class="md-bottom-bar-item" :class="classes" :disabled="disabled" @click="setActive" v-if="href">
-    <md-icon>{{ mdIcon }}</md-icon>
+    
+    <md-icon v-if="mdIcon">{{ mdIcon }}</md-icon>
```
Comment (marcosmoura) [inline src/components/mdBottomBar/mdBottomBarItem.vue]: You don't need those `v-if`. Create just only one `<md-icon>`:
``` vue
<md-icon v-if="mdIcon || mdIconSrc" :md-src="mdIconSrc" :md-iconset="mdIconset">{{ mdIcon }}</md-icon>
```

**Comment by marcosmoura** [comment]
Hi @sombriks

Sorry about the delay to answer that. I got a Dengue here and the things got bad in here.
I liked the idea. I just made some comments over there.

Thank you!

**Comment by sombriks** [comment]
Hey, Dengue is serious thing! Take care.

i'll happily provide the requested changes, :+1: :+1: :+1: 

thanks!

**Pull request merged**